### PR TITLE
Issue 1370

### PR DESCRIPTION
**Pull request by helenemccarron**
Issue #1370

### CODE REVIEW

**Comment by ajluk** [comment]
@helenemccarron in this PR I have set `ProblemContentContainer: (
            display: inline,
        ),
        ProblemParagraphContainer: (
            display: inline,
        ),    `

File: styles/designs/carnival/parts/_eoc-shapes.scss [hunk 1]
```diff
@@ -71,12 +71,13 @@
             _components:(
                 $Eoc_Header,
                 $Exercisetitle_Container,
-                map-merge( $Problem_container,(
-                  _components: (
-                    $Problem_Number,
-                    $Problem_Divider,
-                  )
-                ))
+                map-merge($Problem_wrapper,(
```
Comment (Stackblocks) [inline styles/designs/carnival/parts/_eoc-shapes.scss]: for style purposes, should`$Problem_wrapper` here and `$Problem_container` have their second terms capitalized to be aligned with their peers?

File: styles/designs/carnival/parts/_eoc-shapes.scss [hunk 2]
```diff
@@ -87,7 +88,7 @@
         map-merge($EocContainer,(
             _components:(
                 $Eoc_Header,
-                map-merge( $Problem_container,(
+                map-merge( $Problem_Container,(
```
Comment (ajluk) [inline styles/designs/carnival/parts/_eoc-shapes.scss]: I have checked it locally on chemistry and it wokrs good but in apbiology and biology it doesn't seem to work. I think EocExerciseWithoutSubtitle shape need to be modified to include ProblemWrapper. Now it only has ProblemContainer included. So [data-type="problem"] isn't displaying as a table.

Comment (helenemccarron) [reply styles/designs/carnival/parts/_eoc-shapes.scss]: Great catch! I'll take a look today, thank you!

**Comment by helenemccarron** [comment]
@ajluk I made the changes you requested.

**Pull request merged**